### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by hhadian**
Please take a quick look before merging. I'm not 100% sure about all fixes.

### CODE REVIEW

File: egs/wsj/s5/steps/chaina/get_raw_egs.sh [hunk 1]
```diff
@@ -208,21 +208,21 @@ if [ $stage -le 0 ]; then
        lattice-align-phones --replace-output-symbols=true $latdir/final.mdl \
        "$lats_rspecifier" ark:- \| \
        chain-get-supervision $chain_supervision_all_opts \
-       $dir/misc/tree $dir/misc/${lang}.trans_mdl ark:- ark:- \| \
+       $dir/misc/default.tree $dir/misc/${lang}.trans_mdl ark:- ark:- \| \
```
Comment (danpovey) [inline egs/wsj/s5/steps/chaina/get_raw_egs.sh]: should be ${lang}.tree

**Comment by danpovey** [comment]
looks good except for 1 comment.  Thanks!

**Comment by hhadian** [comment]
Thanks, fixed it.

**Comment by danpovey** [comment]
cool.  I'll probably get back to this script later today or to-morrow; I have a batchnorm-related thing to finish off first (we need an alternative to batchnorm that works when minibatches are systematically mismatched).

**Comment by hhadian** [comment]
Interesting. How mismatched are they? I mean are you assuming they can be from totally different languages or just different databases (from the same language)?

**Comment by danpovey** [comment]
The main concern right now is not just multilingual, but the unadapted vs.
adapted runs of the top model.
Gaofeng already implemented batch renorm, and I'm working on something else.
We need to test them.  I hope it will be done within a few days.  But we
can still work on the scripts.  If nothing else, we could just use
NormalizeComponent for the top network.

**Comment by hhadian** [comment]
OK, thanks for the info.

**Pull request merged**